Project/Fall2020_CSC403_Project/FrmLevel.cs

```diff
@@ -199,14 +199,28 @@ private void FrmLevel_PreviewKeyDown(object sender, PreviewKeyDownEventArgs e)
 
       //sets the visibility of the picture depending on the enemy
       //had to do this since the pic of the enemy is stored seperate from the rest of enemy attributes
-      if(enemy == enemyCheeto){
-        picEnemyCheeto.Visible = false;
+      if(enemy == enemyCheeto)
+      {
+                if (picEnemyCheeto.Visible)
+                {
+                    player.AlterHealth(4);
```

Comment:
Would the AlterHealth() calls be better suited in the FrmBattle class, maybe on its closing function?

Reply:
I figured that it made the most sense to alter health when you dispose of the assets for the enemy, as at that point, it is sure that the enemy is defeated, since you could potentially leave a battle before it is won. As well as this, I was more familiar with how this class functions compared to the FrmBattle class and its functions.

Reply:
I do see the issue in potential scalability of this, however, since nobody is planning on adding any more enemies, and since there are only 3, I think hard coding it for each enemy is not an issue.